packages/shared/src/hooks/post/useMutateComment.ts

```diff
@@ -152,10 +152,15 @@ export const useMutateComment = ({
     isLoading: isCommenting,
     isSuccess,
   } = useMutation<MutateCommentResult, unknown, SubmitComment>(
-    (variables) =>
-      requestMethod(graphqlUrl, mutation, variables, {
+    (variables) => {
+      if (isCommenting || isSuccess) {
+        return Promise.resolve(null);
+      }
```

Comment:
Why this change? It makes sense to add isCommenting but returning empty promise after one success feels like user would not be able to do two comments in a row?

Reply:
They should be able to. This is a safeguard to avoid multiple same comments. However we can maybe add this in a different place to be safer.